packages/dd-trace/src/crashtracking/crashtracker.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+// Load binding first to not import other modules if it throws
+const libdatadog = require('@datadog/libdatadog')
+const binding = libdatadog.load('crashtracker')
+
+const log = require('../log')
+const { URL } = require('url')
+const pkg = require('../../../../package.json')
+
+class Crashtracker {
+  constructor () {
+    this._started = false
+  }
+
+  configure (config) {
+    if (!this._started) return
+
+    try {
+      binding.updateConfig(this._getConfig(config))
+      binding.updateMetadata(this._getMetadata(config))
+    } catch (e) {
+      log.error(e)
+    }
+  }
+
+  start (config) {
+    if (this._started) return this.configure(config)
+
+    this._started = true
+
+    try {
+      binding.initWithReceiver(
```

Comment:
```suggestion
      binding.init(
```
I think this should be init instead?
https://github.com/DataDog/libdatadog-nodejs/blob/main/crates/crashtracker/src/lib.rs#L5

Reply:
Yes, I'll add some more tests as nothing seems to have caught it.

Reply:
Fixed in latest commit along with new tests.